fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -736,6 +736,10 @@ StringRef StringRefOf(const char* s) {
 void SimulationConfig::generateNormalConfig(int minimumReplication, int minimumRegions) {
 	set_config("new");
 	const bool simple = false;  // Set true to simplify simulation configs for easier debugging
+	// generateMachineTeamTestConfig set up the number of servers per machine and the number of machines such that
+	// if we do not remove the surplus server and machine teams, the simulation test will report error.
+	// This is needed to make sure the number of server (and machine) teams is no larger than the desired number.
+	bool generateMachineTeamTestConfig = (BUGGIFY && g_random->random01() < 0.1) ? true : false;
```

Comment:
Hmm, this one has been changed in my branch. Maybe it's because GitHub didn't update the file? Anyway, I will double check GitHub shows the consistent content.